.github/workflows/sdfactorypipeline.yml

```diff
@@ -44,7 +22,7 @@ env:
   # Use GH Container Registry instead of docker
   # github.repository equals to <account|org>/<repo>
   REGISTRY: ghcr.io
-  IMAGE_NAME: catenax-ng/tx-sd-factory/sdfactory
+  IMAGE_NAME: cofinity-x/cofinity-x-sd-factory-core/sdfactory
```

Comment:
Building docker images as an open source project for commercial provider does not seem be right. This might be a critical legal topic, so this should definitely not be merged